tests/acceptance/features/webUISharingInternalGroups/shareWithGroupsEdgeCases.feature

```diff
@@ -32,3 +32,31 @@ Feature: Sharing files and folders with internal groups
       | group     |
       | ?\?@#%@,; |
       | नेपाली    |
+
+  Scenario: Share file with a user and a group with same name
```

Comment:
I think we can add more scenarios like this. Similar tests were recently added in core. Please check them for reference.
https://github.com/owncloud/core/pull/36283